docs/02-get-started/README.md

```diff
@@ -9,9 +9,11 @@ All the steps are performed in the `default` Namespace.
 
 ## Prerequisites
 
-- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (1.19 or greater)
+>**CAUTION:** As of version 1.20, [Kubernetes deprecated Docker](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) as a container runtime in favor of [containerd](https://containerd.io/). Due to a different way in which containerd handles certificate authorities, Kyma's built-in Docker registry does not work correctly on clusters running with a self-signed TLS certificate on top of Kubernetes installation where containerd is used as a container runtime. If that is your case, either upgrade the cluster to use Docker instead of containerd, generate a valid TLS certificate for your Kyma instance or [configure an external Docker registry](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-07-set-external-registry/).
+
```

Comment:
```suggestion
```

I have doubts as to whether we should put this NOTE here. It's Get Started, it's supposed to be fairly concise and quick to do and it doesn't look great to require the user to do some switching before they even started installing Kyma. 
Maybe, for the sake of GSG, we can just use version 1.19 of Kubernetes here? The installation step links to the main Kyma installation doc anyway "for other ways to install Kyma", so the user will be able to see the CAUTION note there.

Optionally, we can link to that installation doc two lines below like this:
```
[k3d](https://k3d.io) (v4.0.0 or higher with [Kubernetes](https://kubernetes.io/docs/setup/) v1.19 - for higher versions, see [this tutorial](../../04-operation-guides/operations/02-install-kyma.md)
```